frontend/webpack.config.ts

```diff
@@ -62,6 +62,7 @@ module.exports = function (_env: any, argv: { hot?: boolean; mode: string | unde
                     ? JSON.stringify('')
                     : JSON.stringify('https://localhost:4000'),
                 'process.env.REACT_APP_BACKEND_PATH': JSON.stringify('/multicloud'),
+		'process.env.REACT_APP_NONKUBERNETES_API_PATH': JSON.stringify('/hub-of-hubs-nonk8s-api'),
```

Comment:
Could we use `REST_API_URL` directly since we already introduced this environment variable in https://github.com/stolostron/hub-of-hubs-console/blob/cd4e135fb65933b34342b6d73d98cb47bf5a139d/backend/src/routes/events.ts#L117?

Reply:
Let us merge this PR firstly. I can help on do modification to use `REST_API_URL`.